src/reporters/tiny.js

```diff
@@ -0,0 +1,66 @@
+import path from "path";
+
+export default report;
+
+async function report(runningTests, startTime) {
+  console.log("");
+  const summary = await reportSequentially(runningTests);
+  const { failed, passed, endTime } = summary;
+  const ok = color.green(`✓ ${passed}`);
+  const notOk = failed ? color.red(`✗ ${failed}`) : "";
+  const runTime = color.grey(`Ran in ${endTime - startTime}ms`);
+  console.log(`${ok} ${notOk} ${runTime}`);
+}
+
+async function reportSequentially(runningTests) {
+  let failed = 0;
+  for await (const result of runningTests) {
+    if (result.error) {
+      failed++;
+      console.log(formatResult(result));
+    }
+  }
+  return { failed, passed: runningTests.length - failed, endTime: Date.now() };
+}
+
+function formatResult(result) {
```

Comment:
since this is only used in to format `error`-s maybe one of `formatError`, `formatFailure`, `formatTestFailure` is a better name?

Reply:
Definitely, formatTestFailure seems more appropriate.